### ISSUE TITLE
without JavaScript, clicking a topic does nothing

### ISSUE THREAD

**Issue by wolftune** [mitchellwrosen/dohaskell#3]
probably should do the same as the arrow, load a permalink page

**Comment by mitchellwrosen** [comment]
Yeah, I'm not too interested in getting the site functioning without JavaScript, at the moment.

**Comment by wolftune** [comment]
no prob, but figured a ticket wouldn't hurt anything

**Comment by mitchellwrosen** [comment]
Absolutely, I'll leave this open for whoever wants to come along and fix it. Might be me, eventually.
